tests/fast-integration/eventing-test/eventing-test-prep.js

```diff
@@ -98,17 +121,6 @@ describe('Eventing tests preparation', function() {
     console.log('Assigning SKR to scenario in Compass');
     // Create a new scenario (systems/formations) in compass for this test
     await addScenarioInCompass(director, scenarioName);
```

Comment:
I understand, but I still would leave it in a separate function, because it's IMO easier to understand and divide the code. The function along with the `addScenarioInCompass` function call contains the logging, it might even be in the future, that we will need to add something else, so the function will be handy. And it the end it's better to have two sall functions than one big.

What do you think ?

Reply:
agree, but then I have a nit comment, that should we move out the debug line out of this function? so that we don't have to pass the `skrinfo` as argument to this method `setupCompassScenario(skrinfo)`. Right now, the `skrinfo` is just used in the debug line and its not really needed the the function for any business logic.

Reply:
Done